packages/support/lib/node.js

```diff
@@ -150,4 +151,19 @@ function getObjectSize (obj) {
   return getCalculator(new WeakSet())(obj);
 }
 
-export { requirePackage, getObjectSize };
+const OBJECTS_MAPPING = new WeakMap();
+
+/**
+ * Calculates a unique object identifier
+ *
+ * @param {object} object Any valid ECMA object
+ * @returns {string} A uuidV4 string that uniquely identifies given object
+ */
+function getObjectId (object) {
+  if (!OBJECTS_MAPPING.has(object)) {
+    OBJECTS_MAPPING.set(object, uuidV4());
```

Comment:
do we care about the size of this object map getting out of hand?

Reply:
it should not because it only contains references. And the map object itself is quite small